### PR TITLE
Awrs 1769

### PR DESCRIPTION
**Pull request by versatil56**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Alcohol Wholesaler Registration Scheme (AWRS) cancellation") {
```
Comment (jakobgrunig) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: Please add the salutation to all templates, like in this other template you have:
https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationEmailRegChange.scala.html#L3

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <p @pStyle>Your request to cancel your AWRS registration for @{params("organisationName")} was received on @{params("submissionDate")}.</p>
+<p @pStyle>What happens next</p>
+    <p @pStyle>Your details on the AWRS register will be changed to show that your business has deregistered on the 12 December 2016.</p>
+    <p @pStyle>Unless you are joining a group or partnership, you must not trade alcohol wholesale after the @{params("submissionDate")}. </p>
+    <p @pStyle>You must sell or dispose of any existing wholesale stock by this date. If you continue to sell alcohol wholesale without an AWRS number you could be liable to criminal prosecution, financial penalties and/or seizure of alcohol stock (whether or not duty paid).</p>
+}
```
Comment (jakobgrunig) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: Please add the signature to all templates.
For example:
https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationEmailRegChange.scala.html#L7

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Alcohol Wholesaler Registration Scheme (AWRS) cancellation") {
+    <p @pStyle>Your request to cancel your AWRS registration for @{params("organisationName")} was received on @{params("submissionDate")}.</p>
+<p @pStyle>What happens next</p>
+    <p @pStyle>Your details on the AWRS register will be changed to show that your business has deregistered on the 12 December 2016.</p>
```
Comment (jakobgrunig) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: Should this be a parametrised date? "12 December 2016"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html (hunk 1 above)
Comment (versatil56) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: On these emails we don't know who we are addressing, therefore, we are unable to add a salutation.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html (hunk 3 above)
Comment (versatil56) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: Thanks for this, the original template had the date hardcoded. I have had a conversaiton with the template creator and he said that it is meant to be submissionDate, now changed.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html (hunk 2 above)
Comment (versatil56) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html]: This has now been done.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+What happens next
+
+Your details on the AWRS register will be changed to show that your business has deregistered on the @{params("submissionDate")}.
+
+Unless you are joining a group or partnership, you must not trade alcohol wholesale after the @{params("submissionDate")}.
+
+You must sell or dispose of any existing wholesale stock by this date. If you continue to sell alcohol wholesale without an AWRS number you could be liable to criminal prosecution, financial penalties and/or seizure of alcohol stock (whether or not duty paid).
+
+
```
Comment (jakobgrunig) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.txt]: Please add the signature to the txt versions too.
EG: https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationConfirmationAPI4.scala.txt#L21

**Pull request merged**